tests/assets/api/addPageAction-unload.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>NEWRELIC-9370: addPageAction Unload Test</title>
+  {init}
+  {config}
+  {loader}
+
+</head>
+
+<body>
+  <div id="content">
+    This page is used to call `addPageAction` immediately followed by a navigation.
+  </div>
+  <a id="btn" href="#btn">Click Me</a>
+  <script>
+    window.triggerPageActionNavigation = function () {
+      newrelic.addPageAction('pageaction', window.location)
```

Comment:
We should probably consider enforcing that objects passed to pageAction API be shallow k:v obj of [string]: string|number|boolean|null|undefined

Reply:
I agree. Something we can take into consideration when redesigning harvest.

Reply:
I put in a placeholder ticket for now: https://issues.newrelic.com/browse/NEWRELIC-9438